saltyrtc/server/__init__.py

```diff
@@ -13,16 +13,18 @@
 from .task import *  # noqa
 from .util import *  # noqa
 
+from . import common, events, exception, message, protocol, server, task, util
```

Comment:
Is this change needed to get mypy to run? Fairly certain there was a reason why I did not do this but it's been a long time.

Reply:
I think so, yes.

Reply:
Actually, it may be a mypy bug. Let me check...

Reply:
Reverted in #123. Let's hope tests pass, locally it seems to work.